clang/test/Preprocessor/cuda-macos-includes.cu

```diff
@@ -1,13 +0,0 @@
-// RUN: %clang -cc1 -fcuda-is-device -isysroot /var/empty \
```

Comment:
Should we keep this test but set the -isysroot to %S/Inputs/MacOSX15.1.sdk ?

Reply:
We don't need it because the `ignoring nonexistent directory` are no longer produced. Those paths get pruned before they reach the HeaderSearch now. The old code unconditionally added those directly, the new code goes through the cc1 arguments and those get checked if they exist before they're added to the HeaderSearch.